docs/software/sciapps/cp2k.md

```diff
@@ -63,22 +63,22 @@ MPS] daemon so that multiple MPI ranks can use the same GPU.
 #!/bin/bash -l
 
 #SBATCH --job-name=cp2k-job
-#SBATCH --time=00:30:00           # (1)
+#SBATCH --time=00:30:00 # (1)!
```

Comment:
Hmm, I guess this is supposed to be https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#stripping-comments? It doesn't seem to be doing the expected thing... it seems to strip the whole line, starting from `#SBATCH ...` already.

Reply:
That's annoying. I'll look to see if it is possible to change this behavior. Or maybe just dropping the additional `#` would work, I'll try.

Reply:
Not using `#` nor `!` does what I wanted. Thanks for spotting the issue!